config.py

```diff
@@ -0,0 +1,25 @@
+"""Flask configuration."""
+from os import environ, path
+from dotenv import load_dotenv
+
+basedir = path.abspath(path.dirname(__file__))
+load_dotenv(path.join(basedir, '.env'))
+
+
+class Config:
+    PATH_TO_INITIAL_DATA = path.join(basedir, 'restaurants.json')
```

Comment:
to my mind, `path.join(basedir, 'restaurants.json')`  - this data is for testing and should be set in `DevConfig`. in production you may use a big JSON

Reply:
fixed